HomeWorks/HW3/LZW/LzwCompressor.cs

```diff
@@ -0,0 +1,52 @@
+namespace LZW;
+
+public static class LzwCompressor
+{
+        /// <summary>
+    /// Compresses the specified input file using the Lempel–Ziv–Welch (LZW) algorithm and writes the result to the output file.
+    /// Prints the compression ratio to the console.
+    /// </summary>
+    /// <param name="inputPath">The path to the input file to compress.</param>
+    /// <param name="outputPath">The path where the compressed output should be written.</param>
+    public static void Compress(string inputPath, string outputPath)
+    {
+        byte[] inputBytes = File.ReadAllBytes(inputPath);
+        int originalSize = inputBytes.Length;
+
+        var dict = new Dictionary<string, int>();
+        for (int i = 0; i < 256; i++)
+            dict[((char)i).ToString()] = i;
```

Comment:
```suggestion
        for (int i = 0; i < 256; i++)
        {
            dict[((char)i).ToString()] = i;
        }
```
Местный стайлгайд предписывает всегда фигурные скобки ставить. Тут и ниже.